src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -67,190 +68,186 @@ async Task<(ImmutableArray<SerializableImportCompletionItem>, StatisticCounter)>
             counter.TotalExtensionMethodsProvided = items.Length;
             counter.Report();
 
-            return items;
+            return (items, counter);
         }
 
-        /// <summary>
-        /// Get the metadata name of all the base types and interfaces this type derived from.
-        /// </summary>
-        private static ImmutableArray<string> GetReceiverTypeNames(ITypeSymbol receiverTypeSymbol)
+        public static async Task<(ImmutableArray<SerializableImportCompletionItem>, StatisticCounter)> GetUnimportedExtensionMethodsInCurrentProcessAsync(
+            Document document,
+            int position,
+            ITypeSymbol receiverTypeSymbol,
+            ISet<string> namespaceInScope,
+            bool forceIndexCreation,
+            CancellationToken cancellationToken)
         {
-            using var _ = PooledHashSet<string>.GetInstance(out var allTypeNamesBuilder);
-            AddNamesForTypeWorker(receiverTypeSymbol, allTypeNamesBuilder);
-            return allTypeNamesBuilder.ToImmutableArray();
+            var counter = new StatisticCounter();
+            var ticks = Environment.TickCount;
 
-            static void AddNamesForTypeWorker(ITypeSymbol receiverTypeSymbol, PooledHashSet<string> builder)
+            // First find symbols of all applicable extension methods.
+            // Workspace's syntax/symbol index is used to avoid iterating every method symbols in the solution.
+            var results = await GetExtensionMethodSymbolsAsync(document, position, receiverTypeSymbol, namespaceInScope,
+                                                               forceIndexCreation, cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
            var results = await GetExtensionMethodSymbolsAsync(
                document, position, receiverTypeSymbol, namespaceInScope, forceIndexCreation, cancellationToken).ConfigureAwait(false);
```

This is a nit, but it's also my preferred style here.  Here's why:
1. no arbitrary indent level.
2. no arbitrary break at some point in teh param list.
3. the params all stay together at one level
4. no line is longer than github width.